validator/core/constants.py

```diff
@@ -48,22 +46,24 @@
 # task stuff
 
 
-HOW_MANY_TASKS_MINIMAL_AT_THE_SAME_TIME = 4
-NUMBER_OF_MINUTES_BETWEEN_SYNTH_TASK_CHECK = 90
+HOW_MANY_TASKS_MINIMAL_AT_THE_SAME_TIME = 3
```

Comment:
What does this mean? The minimum number of tasks that can be running at the same time? dont think that makes sense